### PR TITLE
fix: spacing and heights

### PR DESCRIPTION
**Pull request by kibo-chandradeeptalaha**

### CODE REVIEW

File: __test__/e2e/helper/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { screen, waitFor, within } from '@testing-library/react'
-import { UserEvent } from '@testing-library/user-event/dist/types/setup'
+import { UserEvent } from '@testing-library/user-event/dist/types/setup/setup'
```
Comment (kibo-kevinwatts) [inline __test__/e2e/helper/index.ts:2]: was this an accidental commit? no exports from there

File: __test__/e2e/integration/page-template/MyAccountTemplate/MyAccountTemplate.spec.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { composeStories } from '@storybook/testing-react'
 import { fireEvent, render, screen, waitFor, within } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
-import { UserEvent } from '@testing-library/user-event/dist/types/setup'
+import { UserEvent } from '@testing-library/user-event/dist/types/setup/setup'
```
Comment (kibo-kevinwatts) [inline __test__/e2e/integration/page-template/MyAccountTemplate/MyAccountTemplate.spec.tsx:4]: was this an accidental commit? no exports from there

File: components/layout/Login/LoginContent/LoginContent.spec.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import userEvent from '@testing-library/user-event'
 
 import * as stories from './LoginContent.stories' // import all stories from the stories file
 
-import type { UserEvent } from '@testing-library/user-event/dist/types/setup'
+import type { UserEvent } from '@testing-library/user-event/dist/types/setup/setup'
```
Comment (kibo-kevinwatts) [inline components/layout/Login/LoginContent/LoginContent.spec.tsx:10]: was this an accidental commit? no exports from there

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]
The spacing / design changes look good, but looks like you have unneeded changes to type imports  in test

File: components/layout/Login/LoginContent/LoginContent.spec.tsx (hunk 3 above)
Comment (kibo-chandradeeptalaha) [reply components/layout/Login/LoginContent/LoginContent.spec.tsx:10]: Yes. we need to remove this type

**Review by kibo-kevinwatts — APPROVED** [review]

File: __test__/e2e/helper/index.ts [hunk 4]
```diff
@@ -1,7 +1,6 @@
 import { screen, waitFor, within } from '@testing-library/react'
-import { UserEvent } from '@testing-library/user-event/dist/types/setup'
 
-export const addNewCard = async (user: UserEvent) => {
+export const addNewCard = async (user: any) => {
```
Comment (TriptiVermaXP) [inline __test__/e2e/helper/index.ts]: Why we used any here?

**Review by TriptiVermaXP — APPROVED** [review]

**Pull request merged**